tests/integration/run-locally.sh

```diff
@@ -3,19 +3,21 @@
 set -e
 
 expandpath() {
-  CDPATH= cd -- "$1" && pwd -P
+  CDPATH=cd -- "$1" && pwd -P
 }
 
-SCRIPTPATH=$(expandpath "$(dirname -- $0)")
+SCRIPTPATH=$(expandpath "$(dirname -- "$0")")
 cd "$SCRIPTPATH"
 
-export WORKSPACE="$(expandpath ../../..)/workspace"
+export WORKSPACE
+WORKSPACE="$(expandpath ../../..)/workspace"
 
 git clone https://github.com/sstephenson/bats/ || true
-export PATH=$(pwd)/bats/bin:$(expandpath ../../target/debug):$PATH
+export PATH
+PATH=$(pwd)/bats/bin:$(expandpath ../../target/debug):$PATH
 
-rm -rf $WORKSPACE
-mkdir $WORKSPACE
-cp -ar fixtures/* $WORKSPACE
+rm -rf "$WORKSPACE"
+mkdir "$WORKSPACE"
+cp -aR fixtures/* "$WORKSPACE"
```

Comment:
in linux:  -a, --archive
              same as -dR --preserve=all

in mac:  -a    Same as -pPR options. Preserves structure and attributes of files but not directory structure.
having -ar in mac causes the incompatiblity -r -R metioned problem.

since  `-R, -r, --recursive copy directories recursively`, 

can we just keep `cp -a`